app/views/view_components/preview.html.erb

```diff
@@ -1 +1,5 @@
-<%= render(@render_args[:component], @render_args[:args], &@render_args[:block])%>
+<% if ViewComponent::Base.render_monkey_patch_enabled || Rails.version.to_f >= 6.1 %>
+  <%= render(@render_args[:component], @render_args[:args], &@render_args[:block])%>
+<% else %>
+  <%= render_component(@render_args[:component], &@render_args[:block])%>
```

Comment:
Don't we need to pass the args in here too?

```suggestion
  <%= render_component(@render_args[:component], @render_args[:args], &@render_args[:block])%>
```

Reply:
That `render_component` is `ViewComponent::RenderComponentHelper#render_component` which doesn't support `args` so an error will be raised.

I checked the dynamic values feature and it works as expected

Reply:
no need, this answer is great! Thank you for checking!